browser/js/config/config.js

```diff
@@ -21,49 +21,53 @@ app.factory('Config', function ($http) {
 
 });
 
-app.controller('ConfigCtrl', function ($scope, Config) {
+app.value('viewportOptions', ['640x360', '1024x768', '1280x800', '1680x1050']);
+
+app.value('dayFrequencyOptions', [
+    {label: 'Sun', value: 0},
+    {label: 'Mon', value: 1},
+    {label: 'Tue', value: 2},
+    {label: 'Wed', value: 3},
+    {label: 'Thurs', value: 4},
+    {label: 'Fri', value: 5},
+    {label: 'Sat', value:6}
+]);
+
+app.value('hourFrequencyOptions', [
```

Comment:
@vatteh should we condense this to fewer options as discussed on Tuesday? Instead of giving them options for every hour of the day, I think for the first iteration we agreed to only give them 4 choices.